go.mod

```diff
@@ -8,24 +8,25 @@ require (
 	github.com/filecoin-project/go-dagaggregator-unixfs v0.2.0
 	github.com/filecoin-project/go-indexer-core v0.2.16
 	github.com/filecoin-project/go-legs v0.4.4
-	github.com/frankban/quicktest v1.14.2
+	github.com/frankban/quicktest v1.14.3
 	github.com/gogo/protobuf v1.3.2
 	github.com/gorilla/handlers v1.5.1
 	github.com/gorilla/mux v1.7.4
 	github.com/hashicorp/go-retryablehttp v0.7.1
-	github.com/ipfs/go-cid v0.1.0
+	github.com/ipfs/go-cid v0.2.0
 	github.com/ipfs/go-datastore v0.5.1
 	github.com/ipfs/go-delegated-routing v0.2.2
 	github.com/ipfs/go-ds-leveldb v0.5.0
 	github.com/ipfs/go-ipfs v0.12.1
 	github.com/ipfs/go-log/v2 v2.5.1
-	github.com/ipld/go-ipld-prime v0.16.0
+	github.com/ipld/go-ipld-adl-hamt v0.0.0-20220614114547-20a727441376
+	github.com/ipld/go-ipld-prime v0.16.1-0.20220607093021-c88f0b441e80
```

Comment:
Depends on unreleased commit in `master` that contains `bindnode` improvements/fixes.